### ISSUE TITLE
Formatter class formating issues

### ISSUE THREAD

**Issue by Sorien** [cnpack/cnwizards#123]
```pascal
type
  TFoo = class(TObject)
  private
    procedure At();
    class var
      Faa: TObject;
    property FaaProp: string read GetFaa;
  end;
```

after format
```pascal
type
  TFoo = class(TObject)
  private
    procedure at();
    class var
      Faa: TObject;
      property
      FaaProp: string read GetFaa;
  end;
```
- `At` -> `at`
- property has wrong indention

_version_: 
Version: 1.1.9.991 Build 2020.03.12
Version: 1.2.0.1012_Unstable Build 2020.09.16

**Comment by Sorien** [comment]
fixed by 29a18fe645ebcf339d15982cfee1f4068e9e50f1 and 8c7bf945b11cd6c2d83f50fe5caa2e5a0dcad214

thx for super fast fixes

**Comment by shanzhashu** [comment]
Integrated into 1013 nightly-build version.
